sites/breadcrumbs/inc-breadcrumbs.html

```diff
@@ -1,25 +1,26 @@
-{%- unless page.pageclass contains "home" -%}
+{%- unless page.pageclass contains "home" or page.breadcrumbs == false -%}
 <nav id="wb-bc" property="breadcrumb">
 	<h2>{{ i18nText-breadcrumb }}</h2>
 	<div class="container">
 		<ol class="breadcrumb" typeof="BreadcrumbList">
+{%- if page.breadcrumbs -%}
+	{% assign breadcrumbCounter = 1 %}
+	{% for breadcrumb in page.breadcrumbs %}
+			<li property="itemListElement" typeof="ListItem">
+				<a property="item" typeof="WebPage" href="{{ breadcrumb.link | relative_url }}">
+					<span property="name">{{ breadcrumb.title }}</span>
+				</a>
+				<meta property="position" content="{{ breadcrumbCounter }}">
+			</li>
+			{% assign breadcrumbCounter = breadcrumbCounter | plus:1 %}
+	{%- endfor -%}
+{%- else -%}
 			<li property="itemListElement" typeof="ListItem">
 				<a property="item" typeof="WebPage" href="{{ i18nText-homePage }}">
 					<span property="name">{{ i18nText-home }}</span>
 				</a>
 				<meta property="position" content="1">
 			</li>
-{%- if page.breadcrumbs -%}
-	{% assign breadcrumbCounter = 2 %}
-	{% for breadcrumb in page.breadcrumbs %}
-		<li property="itemListElement" typeof="ListItem">
-			<a property="item" typeof="WebPage" href="{{ breadcrumb.link | relative_url }}">
-				<span property="name">{{ breadcrumb.title }}</span>
-			</a>
-			<meta property="position" content="{{ breadcrumbCounter }}">
-		</li>
-		{% assign breadcrumbCounter = breadcrumbCounter | plus:1 %}
-	{%- endfor -%}
 {% endif %}
 		</ol>
```

Comment:
The goal is to have default first breadcrumb (ex Canada.ca), and then the rest of the breadcrumb items set by individual pages. You will need to flip the logic like how it was.

Reply:
Ended up reverting this part - but kept my extra indentation so that custom breadcrumb stay aligned with the the built-in "Canada.ca" link.